### PR TITLE
44 beneficiary responsible for

### PR DESCRIPTION
**Pull request by dira**

### CODE REVIEW

**Comment by whitmanc** [comment]
Note from meeting: 

* Guardian can be responsible for multiple beneficiaries

File: app/assets/javascripts/app/views/form_view.js.coffee [hunk 1]
```diff
@@ -52,4 +52,4 @@ class App.Views.Form extends Backbone.Form
   _showFirstError: ->
     if (errors = @$('.error')).length > 0
       errors[0].scrollIntoView(true)
-      errors.first().find('input').focus()
+      errors.first().find('input,select').focus()
```
Comment (whitmanc) [inline app/assets/javascripts/app/views/form_view.js.coffee]: Is the `select` necessary as a selector here?

**Comment by whitmanc** [comment]
@dira Everything looks/works good. I only have a single note/question above (https://github.com/accelestate/accelestate/pull/45#discussion_r22869688)

File: app/assets/javascripts/app/views/form_view.js.coffee (hunk 1 above)
Comment (dira) [inline app/assets/javascripts/app/views/form_view.js.coffee]: Yes. If the first error is not an input field, but a select field, it should be focused too. @whitman

**Comment by whitmanc** [comment]
Merged

**Pull request closed without merging**